### ISSUE TITLE
SDRi results were obtained on validation set or test set?

### ISSUE THREAD

**Issue by flyjiang92** [ShiZiqiang/dual-path-RNNs-DPRNNs-based-speech-separation#3]
Hi Dr. Shi, 
Thank you so much for sharing the code! I was wondering that the SDRi of ~19.1 dB was obtained  on the validation set (10 h) or on the test set (5 h), since I have run your well-trained model and could only get ~14.3 dB on the test set. 
Looking forward to your reply. 
Thanks!
Fei

**Comment by ShiZiqiang** [comment]
Hi Fei. Thanks for your message. Please refer to a just closed issue, almost the same problem and solution,:)

**Comment by flyjiang92** [comment]
Oops..I didn't notice that. 
Thank you for your kind reply!
